modules/shimmer/1.0/shimmer.smk

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  N/A
+# Module Author:    Jasper
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+import os
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["shimmer"]`
+CFG = op.setup_module(
+    name = "shimmer",
+    version = "1.0",
+    subdirectories = ["inputs", "shimmer", "outputs"],
+)
+
+# Define rules to be run locally when using a compute cluster
+localrules:
+    _shimmer_input_bam,
+    _shimmer_output_vcf,
+    _shimmer_all
+
+
+##### RULES #####
+
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _shimmer_input_bam:
+    input:
+        bam = CFG["inputs"]["sample_bam"],
+        bai = CFG["inputs"]["sample_bai"]
+    output:
+        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam",
+        bai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bai"
+    run:
+        op.relative_symlink(input.bam, output.bam)
+        op.relative_symlink(input.bai, output.bai)
+
+
+rule _shimmer_on:
```

Comment:
Is it possible to include a rule to build/make Shimmer from the source code you have in `etc`?

Reply:
That should totally be possible.